starter-kits/credential-registry/server/tob-api/api_v2/urls.py

```diff
@@ -43,27 +42,20 @@
     search.CredentialTopicSearchView,
     "Credential Topic Search",
 )
+router.register(r"search/credential", search.CredentialSearchView, "Credential Search")
 router.register(
-    r"search/credential",
-    search.CredentialSearchView,
-    "Credential Search",
-)
-router.register(
-    r"search/autocomplete",
-    search.NameAutocompleteView,
-    "Name Autocomplete",
+    r"search/autocomplete", search.NameAutocompleteView, "Name Autocomplete"
 )
 
 # Misc endpoints
 miscPatterns = [
     path("feedback", misc.send_feedback),
     path("quickload", misc.quickload),
+    path("verify", misc.verify_credential),
```

Comment:
The only change in this file is adding the `verify` path and removing miscpatterns and swaggerpatterns from using format suffixes, as they were causing issues (and we don't need format suffixes on those endpoints).